src/parse_execution.cpp

```diff
@@ -297,7 +297,7 @@ end_execution_reason_t parse_execution_context_t::run_if_statement(
     size_t next_elseif_idx = 0;
 
     // We start with the 'if'.
-    trace_if_enabled(*parser, L"if");
+    rust_trace_if_enabled(*parser, L"if");
```

Comment:
With the above suggestion, all the function call changes in this file will be avoided.